controllers/lesson.js

```diff
@@ -0,0 +1,14 @@
+const lessonService = require('~/services/lesson')
+
+const createLesson = async (req, res) => {
+  const currentUser = req.user
+  const data = req.body
+
+  const newLesson = await lessonService.createLesson(currentUser, data)
```

Comment:
i think here is no need to pass all currentUser data if you use only id from it so you can pass only id of current user

Reply:
fixed